src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -34,7 +34,8 @@ public abstract partial class Workspace : IDisposable
         private readonly HostWorkspaceServices _services;
         private readonly BranchId _primaryBranchId;
 
-        private readonly IWorkspaceOptionService? _workspaceOptionService;
+        // Can be null for tests.
+        private readonly IOptionService? _optionService;
```

Comment:
Can we instead create a mock instance of the interface in the tests and keep this non-null?